2.8

magnum129 opened the pull request:
Don't know if you are interested, but this adds a OctoPrint progress indicator of sorts, via the unused row of pixels along the bottom of the matrix. There is a bit of flicker though :-/

Qrome commented:
Thanks for starting it from the 2.8 branch.  I pointed the PR back to that branch.    I will look at this more.   I think we can do this without the flicker.

Qrome commented:
I have updated the math as the conversion to Float in the math does not work in the new stricter Arduino libs.

```
int numberOfLightPixels = (printerClient.getProgressCompletion().toFloat() / float(100)) * (matrix.width() - 1);
```
I have also updated to be able to enable or disable it from the web interface.

magnum129 commented:
Cool. Thanks for adding it in, and fixing it up!

On Sat, Mar 2, 2019, 9:19 PM Qrome <notifications@github.com> wrote:

> I have updated the math as the conversion to Float in the math does not
> work in the new stricter Arduino libs.
>
> int numberOfLightPixels = (printerClient.getProgressCompletion().toFloat() / float(100)) * (matrix.width() - 1);
>
> I have also updated to be able to enable or disable it from the web
> interface.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Qrome/marquee-scroller/pull/87#issuecomment-468980290>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AXnAyRL7gF0W6_AlK5q3csSzzuoKEwERks5vSzE6gaJpZM4bart3>
> .
>
